src/components/styles/StyledLink.tsx

```diff
@@ -1,32 +1,16 @@
 import { css } from "@emotion/react";
```

Comment:
(placement of comment is random) I might just be making this up but I feel like the spacing between the task log lines shrunk, making them a little harder to read than before

<img width="1422" alt="Screen Shot 2022-10-26 at 8 08 24 PM" src="https://user-images.githubusercontent.com/47064971/198161568-ec4dc97f-6ace-4073-bf2d-3ba87e61c7c6.png">